roles/core/cluster/tasks/cluster.yml

```diff
@@ -74,6 +74,35 @@
     - scale_cluster_profile_name is defined and scale_cluster_profile_name != 'None'
     - scale_cluster_profile_name not in gpfs_cluster_system_profile
 
+- block:
+    - name: check | cluster profile name validation
+      assert:
+        that:
+          - scale_cluster_profile_name is not match("gpfs.*")
```

Comment:
I think this test is wrong, it should be `gpfs*` instead of `gpfs.*`

> A user-defined profile must not begin with the string 'gpfs' and must have the .profile suffix.


**Edit:** after testing, it seems like that `.*` maybe it does catch `gpfs*`, strange and not intuitive.

Reply:
However, I think this test fails though, because in `main.yml` we have the comments: 

> Specifies a predefined profile of attributes to be applied.
> System-defined profiles are located in /usr/lpp/mmfs/profiles/.
> The following system-defined profile names are accepted:
> gpfsprotocoldefaults and gpfsprotocolrandomio
> eg: If you want to apply gpfsprotocoldefaults then specify scale_cluster_profile_name: gpfsprotocoldefaults

So if the user is trying to apply the system profile `gpfsprotocoldefaults`, this would fail here.